src/early/ipfs-script.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs-extra'
+import { join } from 'path'
+import os from 'os'
+import { logger } from '../utils'
+
+export default function () {
+  // Note: during runtime, we only do this for darwin.
+  if (os.platform() !== 'darwin') {
+    return
+  }
+
+  // Ignore during development because the paths are not the same.
+  if (process.env.NODE_ENV === 'development') {
+    logger.info('[ipfs on path] unavailable during development')
+    return
+  }
+
+  // Don't make any changes if IPFS already exists...
+  if (fs.existsSync('/usr/local/bin/ipfs')) {
+    logger.info('[ipfs on path] was not added, already exists')
```

Comment:
For a subsequent PR, we could prompt the user here. 

"you appear to have a version of the `ipfs` command line tool already. Would you like to let ipfs-desktop replace it with the a version that it can keep up to date?"

...that kind of thing.

Reply:
Yes, that's the goal. I'm just making this one as simple as possible.